web/modules/custom/do_base/do_base.deploy.php

```
@@ -1369,6 +1420,98 @@ function _do_base_remove_related_list(NodeInterface $node): void {
+ */
+function _do_base_image_list_paragraph(): ParagraphInterface {
+  $paragraph = Paragraph::create([
+    'type' => 'image_list',
+    'uuid' => _do_base_image_list_uuid(),
+    'field_c_p_theme' => 'light',
+    // The blog list below already carries a tinted background, so this band is
+    // left plain to keep the homepage alternating rather than running two
+    // tinted sections together.
+    'field_c_p_background' => FALSE,
+    'field_c_p_vertical_spacing' => 'both',
+  ]);
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Drupal\paragraphs\Entity\Paragraph' in method '_do_base_image_list_paragraph'. (undefined)

(StaticAccess)